finder/java/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.flutter.appiumfinder</groupId>
+    <artifactId>appium-flutterfinder-java</artifactId>
+    <version>0.4.4</version>
+    <packaging>jar</packaging>
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>Flutter finder plugin for Appium in java</description>
+    <url>https://github.com/testsigma-eng/appium-flutterfinder-java</url>
```

Comment:
Then, is it better to leave a link to this repository in the README of https://github.com/appium-userland/appium-flutter-driver/tree/main/finder/kotlin instead of adding this to this repository for you?

It looks like you work for this company? and move the repository to this repository...? Sorry, I'm not sure the relation ship of this pom's information and you.

Actually the repository is MIT, so theoretically they can be copied with their copyright, but it seems like its better to have a link to their repository in https://github.com/appium-userland/appium-flutter-driver/tree/main/finder/kotlin as README instead of copying them into this repository